.storybook/main.js

```diff
@@ -15,6 +15,14 @@ module.exports = {
   },
   staticDirs: ['../static'],
   webpackFinal: async (config) => {
+    // Transpile Gatsby module because Gatsby includes un-transpiled ES6 code.
+    config.module.rules[0].exclude = [/node_modules\/(?!(gatsby)\/)/]
+
+    // Use babel-plugin-remove-graphql-queries to remove static queries from components when rendering in storybook
+    config.module.rules[0].use[0].options.plugins.push(
+      require.resolve('babel-plugin-remove-graphql-queries')
+    )
```

Comment:
From https://www.gatsbyjs.com/docs/how-to/testing/visual-testing-with-storybook/#manual-configuration.

Reply:
Nicee!